### DISCUSSION TITLE
Can the versioning feature be used without using manifests?

### DISCUSSION THREAD

**Discussion by aggieNick02** [microsoft/vcpkg#20692] [Q&A] [answered]
Is there a way to specify versions without using manifests? All of the documentation for versioning is how to do it with manifests, but manifests still have some significant shortcomings that prevent their usage:
https://github.com/microsoft/vcpkg/issues/20690
https://github.com/microsoft/vcpkg/issues/12289

**Comment by autoantwort** [comment, marked as answer]
No, versioning is only possible with manifest. But you can checkout an old version of vcpkg when you need an old version of a lib.

**Reply by aggieNick02** [reply]
Thanks for answering.
